tarofreighter/chain_porter.go

```diff
@@ -779,13 +693,20 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 
 		// Construct our template PSBT to commits to the set of dummy
 		// locators we use to make fee estimation work.
-		sendPacket, err := taroscript.CreateTemplatePsbt(
-			currentPkg.SendDelta.Locators,
+		sendPacket, err := taroscript.CreateAnchorTx(
+			currentPkg.VirtualPacket.Outputs,
 		)
 		if err != nil {
 			return nil, err
 		}
 
+		sendPacket.Outputs[0].TaprootInternalKey = schnorr.SerializePubKey(
```

Comment:
Any reason this can't be done within the function above?

Reply:
This was cleaned up in the current PRs.